src/clients/ProfitClient.ts

```diff
@@ -590,6 +590,8 @@ export class ProfitClient {
       exclusivityDeadline: 0,
       exclusiveRelayer: ZERO_ADDRESS,
       message: EMPTY_MESSAGE,
+      fromLiteChain: false, // TODO: We *may* care about this in the future.
+      toLiteChain: false, // TODO: We *may* care about this in the future.
```

Comment:
These don't affect gas consumption so we're OK to drop the comments.

```suggestion
      fromLiteChain: false,
      toLiteChain: false,
```